deckhouse-controller/pkg/controller/deckhouse-release/controller.go

```diff
@@ -456,11 +474,17 @@ func (r *deckhouseReleaseReconciler) tagUpdate(ctx context.Context, pods []corev
 		return fmt.Errorf("registry (%s) client init failed: %s", repo, err)
 	}
 
+	r.metricStorage.CounterAdd("deckhouse_registry_check_total", 1, map[string]string{})
+	r.metricStorage.CounterAdd("deckhouse_kube_image_digest_check_total", 1, map[string]string{})
+
 	repoDigest, err := regClient.Digest(tag)
 	if err != nil {
+		r.metricStorage.CounterAdd("deckhouse_registry_check_errors_total", 1, map[string]string{})
 		return fmt.Errorf("registry (%s) get digest failed: %s", repo, err)
 	}
 
+	r.metricStorage.GaugeSet("deckhouse_kube_image_digest_check_success", 1.0, map[string]string{})
+
```

Comment:
not sure, but maybe it should've been of counter type? I mean this Gauge is going to be set to 1 once and for all..

Reply:
Fixed